src/main/java/gov/cabinetofice/gapuserservice/web/DepartmentController.java

```diff
@@ -23,6 +24,9 @@ public class DepartmentController {
     private final RoleService roleService;
     private final DepartmentService departmentService;
 
+    @Autowired
```

Comment:
Because of some spring magic you actually dont need this annotation. I believe its due to @RequiredArgsConstructor generating a constructor with this final var as a param, spring knows to inject the relevant bean

Reply:
amazin